.gitignore

```diff
@@ -13,3 +13,5 @@ typedoc
 .DS_Store
 .nyc_output
 npm-debug.log
+dist/
+.idea
```

Comment:
Nit: arguably, editor-specific ignores should be in the user's global git ignore.

Reply:
Agreed, this slipped in by accident